lms/djangoapps/verify_student/api.py

```diff
@@ -33,3 +34,42 @@ def send_approval_email(attempt):
     else:
         email_context = {'user': attempt.user, 'expiration_datetime': expiration_datetime.strftime("%m/%d/%Y")}
         send_verification_approved_email(context=email_context)
+
+
+def create_verification_attempt(user, name, status, expiration_datetime=None):
+    """
+    Create a verification attempt.
+
+    Args:
+        user (User): the user (usually a learner) performing the verfication attempt
+        name (string): the name of the user
```

Comment:
This should be the name being ID verified. It's not necessarily the current user's name.